### PR TITLE
add event for detail comic screen

### PR DESCRIPTION
**Pull request by anhmv-1205**
#### Screenshots
Have not

### CODE REVIEW

**Comment by anhmv-1205** [comment]
ready

File: app/src/main/java/com/sun_asterisk/comics_01/screen/comic/ComicDetailActivity.java [hunk 1]
```diff
@@ -97,10 +106,18 @@ private void getRemoteData() {
...
 
     @Override
     public void onError(Exception exception) {
+        if (exception != null) mProgressBar.setVisibility(View.GONE);
+    }
+
+    @Override
+    public void onItemClickListener(Chapter chapter) {
+        Intent intent = ReadComicActivity.getReadComicIntent(this, chapter.getImagesLink());
+        startActivity(intent);
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/comics_01/screen/comic/ComicDetailActivity.java]:
```suggestion
        startActivity(ReadComicActivity.getReadComicIntent(this, chapter.getImagesLink()));
```

File: app/src/main/java/com/sun_asterisk/comics_01/screen/read/ReadComicActivity.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_read_comic);
+        receiveData();
+    }
+
+    private void receiveData() {
+        Intent intent = getIntent();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/comics_01/screen/read/ReadComicActivity.java:29]: check null intent

**Comment by anhmv-1205** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -12,7 +12,12 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".screen.comic.ComicDetailActivity"
+        <activity
+            android:name=".screen.read.ReadComicActivity"
+            android:screenOrientation="portrait">
```
Comment (tuanlvt-1117) [inline app/src/main/AndroidManifest.xml]:
```suggestion
            android:screenOrientation="portrait"/>
```

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -12,7 +12,12 @@
...
+        </activity>
+        <activity
+            android:name=".screen.comic.ComicDetailActivity"
             android:screenOrientation="portrait">
```
Comment (tuanlvt-1117) [inline app/src/main/AndroidManifest.xml]:
```suggestion
            android:screenOrientation="portrait"/>
```

**Comment by anhmv-1205** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**